appium/tests/screenobjects/new-message.screen.ts

```diff
@@ -216,8 +217,12 @@ class NewMessageScreen extends BaseScreen {
     await this.showRecipientInputIfNeeded();
     const recipientCell = await $(`~aid-${type}-${order}-label`);
     await ElementHelper.waitElementVisible(recipientCell);
-    const name = await recipientCell.getValue();
-    expect(name).toEqual(`  ${recipient}  `);
+    await recipientCell.waitUntil(async  function () {
+      return (await recipientCell.getValue() === `  ${recipient}  `)
+    }, {
+      timeout: 15,
+      timeoutMsg: 'expected text to be different after 15s'
+    });
```

Comment:
Should use the new method here

Reply:
do you mean another method?

Reply:
You've just implemented a `waitForValue` method. Why not use it here?